.github/workflows/modular-arch-quality-gates.yml

```
@@ -0,0 +1,971 @@
+          else
+            echo "🔍 Running quality gates with full notifications for ${{ matrix.module }}"
+          fi
+
+      - name: Setup Node.js
+        if: steps.check-config.outputs.silent_mode != 'true'
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      
+      - name: Setup Go
+        if: steps.check-config.outputs.silent_mode != 'true' && hashFiles('frontend/packages/${{ matrix.module }}/bff/go.mod') != ''
```

actionlint (1.7.7)

if: condition "steps.check-config.outputs.silent_mode != 'true' && hashFiles('frontend/packages/${{ matrix.module }}/bff/go.mod') != ''" is always evaluated to true because extra characters are around ${{ }}

(if-cond)